src/server/routes/rest.js

```diff
@@ -65,6 +66,18 @@ router.get('/gene-query', (req, res) => {
   });
 });
 
+//expose a rest endpoint for enrichment
+router.get('/enrichment/', (req, res) => {
+  const genes = req.query.genes;
+  let user_settings = {};
+  if (req.query.setting !== undefined) {
+    user_settings = JSON.parse(req.query.setting);
```

Comment:
- use camel case for variables
- how many settings are there?
- generally you should be accessing each setting via it's specific key instead of a an object.

Reply:
Could we elide the trailing slash?  It might not make a difference for Express, but it can for some routers and I'd rather keep things simple and consistent.

Reply:
Also the names of arguments should be meaningful.  `setting` doesn't mean anything.  It would be better to use something like `options`.